kakarot_scripts/aws_lambda/relayers_observer/relayers.json

```diff
@@ -0,0 +1,92 @@
+[
+  {
+    "address": 950135147883551002471284612127692562371997031991686973344811456814871551703
```

Comment:
is this used somewhere else in the codebase @greged93? are we using the relayers in kakarot repo somehow?

Reply:
we are not, they were posted just for tracking